src/Data/List/Membership/Setoid.agda

```diff
@@ -32,14 +32,20 @@ x ∈ xs = Any (x ≈_) xs
 _∉_ : A → List A → Set _
 x ∉ xs = ¬ x ∈ xs
 
+∃∈ : ∀ {p} (P : Pred A p) → Pred (List A) _
+∃∈ P xs = ∃ λ x → x ∈ xs × P x
```

Comment:
I'm going to be deeply unhelpful here and say that I don't really like these names without suggesting a concrete replacement. 

To me these read `Exists in P xs` and `Forall in P xs` neither of which makes sense to me... 

However, as I've said I can't come up with good alternatives. Anyone else have any good ideas?

Reply:
So...:
* the already-existing proofs `∃∈-Any` and `Any-∃∈` introduce the symbolism, so these definitions are exactly the back-formation of the associated operations, given those lemma names...
* I too wondered if they weren't so memorable/mnemonic, but see above: I didn't feel like deprecating the pre-existing names!

That said: does changing the parametrisation to 
```agda
∈ xs P = ∃ λ x → x ∈ xs × P x
```
improve matters? But even then, it can't be defined as a predicate transformer... because of the re-parametrisation!

Reply:
I too read these as `Exists in P`. `∃∈×Pred` ? `∃∈P` ? `∃∈×P` ? ``∃-∈×P` ?

The flipped parametrisation is mildly nicer - but I agree with @MatthewDaggitt that the name needs work. Even if it means introducing a seeming inconsistency with existing naming.

Reply:
Will continue to ponder the alternatives. Thanks for the additional suggestions!

Reply:
First attempt incoming:
* keeps the names you don't like, but with `syntax` as a suffix
* introduces a more reasonable `syntax` for it

This gives us additional options for a 'real' name for the combinators, with the names I introduced available 'solely for the sake of the syntax'...?

Reply:
At this stage, no attempt to propagate use of the new `syntax` to eg `Data.Membership.*.Properties`...

Reply:
NB. These 'modalities' generalise `Diamond` and `Box` from `Relation.Unary.Closure.Base` so perhaps `◇∈` and `□∈` would at least be 'neutral' (if still as incomprehensible with additional `syntax`...?)

Reply:
Certainly equally incomprehensible, but I can definitely see the appeal!

I'm worried about the over-use of `syntax`: it doesn't feel like it is a "fully designed feature", and certainly has no theoretical underpinning. That's a scary one-two.

Reply:
Well, I tend to agree with you regarding `syntax`, and that's why we have this horrible duplication fandango, so that such declarations can be hidden from `import`... but here, I think the syntax does communicate 'best' the intention of the combinator...

Reply:
Incoming commits pick my final proposed name. Please continue to comment, and/or change as you see fit... it's a case of search-and-replace now...